Source/WebCore/rendering/RenderVideo.cpp

```diff
@@ -117,8 +117,14 @@ bool RenderVideo::updateIntrinsicSize()
 
 LayoutSize RenderVideo::calculateIntrinsicSize()
 {
-    if (shouldApplySizeContainment())
-        return LayoutSize();
+    if (shouldApplySizeContainment()) {
+        LayoutSize size;
+        if (auto width = explicitIntrinsicInnerWidth())
+            size.setWidth(width.value());
+        if (auto height = explicitIntrinsicInnerHeight())
+            size.setHeight(height.value());
+        return size;
```

Comment:
Is this needed since we have RenderReplaced:: intrinsicSize? If yes, can we share the code?

Reply:
I'm not quite sure what does this mean?

Reply:
RenderVideo is a RenderReplaced, so why is the RenderReplaced:: intrinsicSize change not enough?

Reply:
OK, makes sense. I don't quite remember, is this to solve some issue or not.  Let me try to remove it.

Reply:
It seems to work! Can you adjust the commit message to explain this better?:
 (WebCore::RenderVideo::calculateIntrinsicSize): Ditto.
I think github allows to edit the commit message by hand...

Reply:
Sure